.github/secret_scanning.yml

```
@@ -29,3 +29,5 @@ paths-ignore:
   - "src/workerd/api/node/tests/fixtures/rsa_pss_private_2048.pem"
   - "src/workerd/api/node/tests/fixtures/x448_private.pem"
   - "src/workerd/api/node/tests/fixtures/tls-nodejs-tcp-server.pem"
+  - "src/workerd/api/tests/startls-server.pem"
+  - "src/workerd/api/tests/starttls-try-server.js"
```

This path needs to change
```suggestion
  - "src/workerd/api/tests/starttls-nodejs-server.js"
```